app/app.go

```diff
@@ -505,6 +505,7 @@ func NewpStakeApp(
 		app.EpochsKeeper,
 		app.ICAControllerKeeper,
 		app.IBCKeeper, // TODO: Move to module interface
+		app.TransferKeeper,
```

Comment:
why do you need this now?

Reply:
need it to get the denom trace of the ibc token:

`denomTrace, found := k.ibcTransferKeeper.GetDenomTrace(ctx, hexBytes)`